sw/device/lib/testing/json/provisioning_data.h

```diff
@@ -91,17 +93,16 @@ UJSON_SERDE_STRUCT(ManufRmaTokenPersoDataOut, \
 // clang-format on
 
 /**
- * Data exported during device (attestation) certificate personalization.
- *
- * TODO(#19455): replace the certificate fields with actual X.509 certificates
- * instead of just the public keys.
+ * Certificates exported during device personalization.
  */
 // clang-format off
 #define STRUCT_MANUF_CERT_PERSO_DATA_OUT(field, string) \
     field(uds_certificate, uint8_t, 596) \
```

Comment:
Is the plan to export the UDS or the TBS_UBS? Or is it exporting the UDS just for testing at the moment?

Reply:
ah yes, I was just testing, thanks for the catch. I updated to print out just the UDS TBS cert.